mandatory/2-function-creation.js

```diff
@@ -15,26 +23,63 @@ function isAcceptableUser(userAge, isLoggedIn) {}
     is applieds and 142.5 should be returned)
 */
 
-function applyDiscount(totalPrice) {}
+function applyDiscount(totalPrice) {
+  let fullDiscount = (totalPrice * 10)/100;
+  let smallDiscount = (totalPrice * 5)/100;
+  if (totalPrice >= 200){
+  return newPrice = totalPrice - fullDiscount;
```

Comment:
looks good! 
the only small thing I would mention here (that won't really make much difference but could be useful to think about) is that smallDiscount is calculated/defined but never used unless the totalPrice is less than 200 - maybe it should go in the if blocks like 
```
if (totalPrice >= 200){
  let fullDiscount = (totalPrice * 10)/100;
  return newPrice = totalPrice - fullDiscount;
} else {
  let smallDiscount = (totalPrice * 5)/100;
  return newPrice = totalPrice - smallDiscount;
  }
}
  return newPrice = totalPrice - fullDiscount;
```

this just means we aren't calculating anything we don't need so it will be a bit more efficient!